src/lib.rs

```diff
@@ -12,6 +12,10 @@
 //! If you want to use a hash table with [amortized resizes](https://github.com/jonhoo/griddle/),
 //! set the `amortize` feature.
 //!
+//! (De)serialization via serde is also available with the `serde` feature.
+//! Serialization note: if the incoming data has repeated keys, the resulting `HashBag` will merge
```

Comment:
Would you mind adding a test for this behavior too?

Reply:
I added two tests to mirror the other tests. I also corrected my wording here. We are not concerned about communicating how we serialize a `HashBag`, only how we deserialize one.